data/contributors/organizations.json

```
@@ -580,5 +580,53 @@
         "last_contribution": "2025-11-19"
       }
     }
+  },
+  {
+    "name": "Camptocamp",
+    "subtitle": "Innovative solutions by Open Source experts",
+    "image": "/images/logo-with-text.svg",
+    "width": "350px",
+    "url": "https://camptocamp.com",
+    "description": "Camptocamp is an active contributor to the QGIS ecosystem—our engineers provide code and bug fixes to QGIS core and server, develop and maintain plugins, integrations and deployment tooling, and contribute documentation, translations and CI. We also help grow and professionalize QGIS adoption by offering training, long-term support and consulting and by participating in community events.",
+    "members": [
+      {
+        "username": "ismailsunni",
+        "author_names": "Ismail Sunni",
+        "from": "2020-04-01",
+        "to": null
+      },
+      {
+        "username": "arnaud-morvan",
+        "author_names": "Arnaud Morvan",
+        "from": "2014-01-03",
+        "to": null
+      },
+      {
+        "username": "yjacolin",
+        "author_names": "Yves Jacolin",
+        "from": "2013-09-13",
+        "to": "2018-12-13"
+     },
+     {
+        "username": "sbrunner",
+        "author_names": "Stéphane Brunner",
+        "from": "2013-07-23",
+        "to": "2022-12-08"
+     }
+    ],
+    "contributions": {
+      "documentation": {
+        "commits": 504,
```

```suggestion
        "commits": 0,
```

This will be automatically updated. See: https://qgis.org/community/contributors/add-organisation/#field-guide